js/ui/codeView.js

```diff
@@ -1484,7 +1498,7 @@ var CodeViewManager = GObject.registerClass({
         // HackUnlock and HackToolbox are inside the com.hack_computer.Clubhouse flatpak
         // and have the Clubhouse appInfo, so we should ignore those cases here
         // to be able to show the HackUnlock and HackToolbox windows
-        } else if (gtkId !== 'com.hack_computer.HackUnlock' && gtkId !== 'com.hack_computer.HackToolbox') {
+        } else if (gtkId !== 'com.hack_computer.HackUnlock' && !gtkId.endsWith('HackToolbox')) {
```

Comment:
This seems rather fragile. What about checking for both the old and the new ids?

Reply:
Okay, I've updated this to check for both ids.